lib/usbmux/index.js

```diff
@@ -51,6 +53,14 @@ function swap16 (val) {
  */
 async function getDefaultSocket (socketPath = DEFAULT_USBMUXD_SOCKET, timeout = 5000) {
   if (!await fs.exists(socketPath)) {
+    if (process.platform === 'linux' && os.release().toLowerCase().includes('microsoft')) {
```

Comment:
The check could be broadened to also check for Windows, after which the code would also run natively on Windows.